airbase/pom.xml

```diff
@@ -225,6 +225,9 @@
         <air.repository.release.id>ossrh</air.repository.release.id>
         <air.repository.release.name>Sonatype Release Snapshots</air.repository.release.name>
         <air.repository.release.url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</air.repository.release.url>
+
+        <!-- ${argLine} is defined by Jacoco, but it can be disabled, so define an empty default here -->
+        <argLine />
```

Comment:
Can this be in a profile?

Reply:
This is required by surefire which is always enabled. Do you see other benefits for moving it to a profile?

Reply:
Wi thought we can define `argLine` when JaCoCo is not loaded (in a profile), and let JaCoCo define argLine when it is loaded (in a profile).

Reply:
Great idea, we shouldn't be worried about compatibility with Jacoco this way. Done and it works!